pom.xml

```diff
@@ -0,0 +1,226 @@
+        <groupId>com.saucelabs</groupId>
+        <artifactId>sauce_testng</artifactId>
+        <version>${sauce_testng.version}</version>         
+      </dependency>
+      <dependency>
+        <groupId>org.testng</groupId>
+        <artifactId>testng</artifactId>
+        <version>${testng.version}</version>      
+      </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-all</artifactId>
+        <version>${mockito-all.version}</version>
+      </dependency>          
+      
+    </dependencies> 
+  </dependencyManagement>
+  
+  <dependencies>               
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>      
+    </dependency>
+    <dependency>      
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>      
+      <scope>test</scope>
+    </dependency>    
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>            
+    </dependency>
+    <dependency>
+      <groupId>com.saucelabs</groupId>
+      <artifactId>sauce_testng</artifactId>           
+    </dependency>               
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>      
+    </dependency>
+    
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>      
+    </dependency>                       
+       
+  </dependencies>
+  <build>    
+    <!--    <finalName>web-test-framework</finalName>-->
+    <plugins>            
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
+        </configuration>
+      </plugin>
+            
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven-surefire-plugin.version}</version>
+        <configuration>          
+          <suiteXmlFiles>
+            <suiteXmlFile>${project.build.testOutputDirectory}/testng-unit.xml</suiteXmlFile>             
+          </suiteXmlFiles>                                       
+          <systemPropertyVariables>                                                 
+            <saucelabs.username>${saucelabs.username}</saucelabs.username>
+            <saucelabs.accessKey>${saucelabs.accessKey}</saucelabs.accessKey>                      
+            <logFilePath>${logFilePath}</logFilePath>                                                              
+          </systemPropertyVariables>          
+        </configuration>
+      </plugin>
+              
+    </plugins>      
+  </build>   
+     
+  <profiles>
+     
+    
+    <profile> 
+      <id>release</id>
+<!--      <activation>

```

Comment:
I recommend not checking in commented out code and leave comment blocks for strictly comments.